src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -209,6 +209,34 @@
 helps['deployment'] = """
 type: group
 short-summary: Manage Azure Resource Manager deployments at subscription scope.
+long-summary: >
+    Upcoming breaking change: a new parameter "scope-type" will be introduced to commands in this group and will be mandatory.
```

Comment:
Can you use a special sign to make the breaking change more conspicuous? e.g. [UPCOMING BREAKING CHANGE] or something else.

Reply:
sounds good. I made it like [**Upcoming breaking change**]